cmd/manager/main.go

```diff
@@ -1,121 +0,0 @@
-package main
```

Comment:
What happened to this class ? why is it deleted ?

Reply:
pushed the wrong branch. fixed.